CONTRIBUTING.md

```diff
@@ -82,7 +70,7 @@ amtool alert add alertname=oops service="my-service" severity=warning \
     --generator-url="http://prometheus.int.example.net"
 ```
 
-or using alertmanager's HTTP API,
+or using [Alertmanager's HTTP API][Alertmanager API browser],
```

Comment:
Don't we have an official documentation for this?

Reply:
It exists, but it is very narrow:
https://www.prometheus.io/docs/alerting/latest/management_api/